test/functional/apps/visualize/_chart_types.js

```diff
@@ -26,7 +26,7 @@ export default function ({ getService, getPageObjects }) {
   describe('chart types', function () {
     before(function () {
       log.debug('navigateToApp visualize');
-      return PageObjects.common.navigateToUrl('visualize', 'new');
+      return PageObjects.visualize.navigateToNewVisualization();
```

Comment:
is it an IE workaround? should we revert it when the issue will be fixed?

Reply:
If I recall correctly, I made this change because this method also implements the work-around to move the mouse, but that issue turned out to be flaky.